### PR TITLE
chapter_computer-vision full scan

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

File: chapter_computer-vision/bounding-box.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Next, we will load the sample images that will be used in this section. We can s
 ```{.python .input}
 d2l.set_figsize((3.5, 2.5))
 img = image.imread('../img/catdog.jpg').asnumpy()
-d2l.plt.imshow(img); 
+d2l.plt.imshow(img)
```
Comment (astonzhang) [inline chapter_computer-vision/bounding-box.md]: Do not remove ; at the end of plot function. Fix all the other cases if any

File: chapter_computer-vision/fcn.md [hunk 2]
```diff
@@ -86,7 +85,8 @@ def bilinear_kernel(in_channels, out_channels, kernel_size):
         center = factor - 1
     else:
         center = factor - 0.5
-    og = np.arange(kernel_size).reshape(-1, 1), np.arange(kernel_size).reshape(1, -1)
+    og = np.arange(kernel_size
+                   ).reshape(-1, 1), np.arange(kernel_size).reshape(1, -1)
```
Comment (astonzhang) [inline chapter_computer-vision/fcn.md]: do not break like this

File: chapter_computer-vision/bounding-box.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_computer-vision/bounding-box.md]: np

File: chapter_computer-vision/fcn.md (hunk 2 above)
Comment (goldmermaid) [reply chapter_computer-vision/fcn.md]: np

File: chapter_computer-vision/fcn.md [hunk 3]
```diff
@@ -86,7 +85,8 @@ def bilinear_kernel(in_channels, out_channels, kernel_size):
         center = factor - 1
     else:
         center = factor - 0.5
-    og = np.arange(kernel_size).reshape(-1, 1), np.arange(kernel_size).reshape(1, -1)
+    og = np.arange(kernel_size).reshape(-1, 1), \
+        np.arange(kernel_size).reshape(1, -1)
     filt = (1 - np.abs(og[0] - center) / factor) * \
            (1 - np.abs(og[1] - center) / factor)
```
Comment (astonzhang) [inline chapter_computer-vision/fcn.md]: todo

File: chapter_computer-vision/kaggle-gluon-dog.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ First, import the packages or modules required for the competition.
 import collections
 import d2l
 import math
-from mxnet import autograd, gluon, init, np, npx
+from mxnet import autograd, gluon, init, npx
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-gluon-dog.md]: my todo

File: chapter_computer-vision/semantic-segmentation-and-dataset.md [hunk 5]
```diff
@@ -196,12 +198,12 @@ for X, Y in train_iter:
     break
 ```
 
-## Put All Things Together
+### Putting Things Together
```
Comment (astonzhang) [inline chapter_computer-vision/semantic-segmentation-and-dataset.md]: my todo, putting all

File: chapter_computer-vision/ssd.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ widely used. Some of the design concepts and implementation details of this
 model are also applicable to other object detection models.
 
 
-## Model
+## SSD Model
```
Comment (astonzhang) [inline chapter_computer-vision/ssd.md]: my todo, the SSD model

**Pull request merged**